fluss-server/src/main/java/com/alibaba/fluss/server/kv/KvTablet.java

```diff
@@ -267,12 +270,31 @@ public LogAppendInfo putAsLeader(
                                 KvRecordReadContext.createReadContext(kvFormat, fieldTypes);
                         ValueDecoder valueDecoder =
                                 new ValueDecoder(readContext.getRowDecoder(schemaId));
-
+                        RowMergeEngine rowMergeEngine = getRowMergeEngine(schema);
+                        /**
+                         * In theory, once configured, it can be ignored directly. In the future, we
+                         * can provide configurations to choose between ignoring or throwing
+                         * exceptions directly.
+                         */
+                        if (partialUpdater != null && rowMergeEngine.ignorePartialUpdate()) {
```

Comment:
Don't ignore partial update, if merge engine don't support partial update, throw exception directly instead of just ignoring it. It's not a expected behavior, ignoring it will hide the exception and make user mistake the rows are written to server..

Reply:
I have always believed that the merge engine should manage what to ignore, which RowKinds to support, and what field types the version column should accept within its own implementation class. If we leave these judgments to the caller, as the types of merge engines increase, the number of conditions will also grow, leading to decreased code readability. Therefore, I still think we should retain 'ignoreDelete' and 'ignorePartialUpdate', or move PartialUpdate validation to the merge engine implementation.